docs/docs/themes/authoring-themes.md

```diff
@@ -0,0 +1,129 @@
+---
+title: How to author a theme
+---
+
+## What’s contained in this tutorial?
+
+By the end of this tutorial, you’ll have done the following:
+
+- set up your environment for theme development
+- built a Gatsby Theme
+
+## Prerequisites (if any)
+
+Before following this tutorial, you should have experience building a Gatsby site.
+
+> ⚠⚠ Gatsby Themes are currently experimental ⚠⚠
```

Comment:
Themes are moving past being experimental, so we can probably leave this out. What do you think, @gatsbyjs/themes-core?